### ISSUE TITLE
Issue loading AHF catalogs and DummyHalo

### ISSUE THREAD

**Issue by robmost** [pynbody/pynbody#425]
I'm trying to load a TIPSY simulation with its corresponding AHF results, all in the same directory. When I load the information in pynbody I get a DummyHalo SimSnap (only the catalog information, with no access to the particle information of the halos). However, when I use AHF files coming from an older version of AHF for the same simulation I do get a Halo SimSnap, with their particle information. I'm trying to do the following: 

```
import pynbody

simPath = ('./simTest.001024') # A TIPSY simulation. 
 
# In the same folder I have simTest.001024.HI, .HII, .iord, ... files and
# simTest.001024.z0.000.AHF_halos, .AHF_profiles, ... files

sim = pynbody.load(simPath)
sim.physical_units()

h = sim.halos()
halo = h[1]

halo_faceon_transf = pynbody.analysis.angmom.faceon(halo)
```
And I get the following error:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-18-c959882a8ace> in <module>()
     35 
---> 36 halo_faceon_transf = pynbody.analysis.angmom.faceon(halo)
     37 print('>>>Done!')

~/anaconda3/lib/python3.6/site-packages/pynbody/analysis/angmom.py in faceon(h, **kwargs)
    155     """
    156 
--> 157     return sideon(h, calc_faceon_matrix, **kwargs)

~/anaconda3/lib/python3.6/site-packages/pynbody/analysis/angmom.py in sideon(h, vec_to_xform, cen_size, disk_size, cen, vcen, move_all, **kwargs)
    104 
    105     if move_all:
--> 106         top = h.ancestor
    107     else:
    108         top = h

AttributeError: 'DummyHalo' object has no attribute 'ancestor'

```

The AHF files that I have to use come from a new version of AHF which gives a few extra columns, different '#ID' convention, etc... It seems that there's a problem when linking the catalog information provided in these AHF files with the particle information.  If I access the particle information of the simulation, i.e. sim['pos'], it works, but not for the halo, i.e. halo['pos'].  Why is this happening? I tried to check the source code but I couldn't find the issue.

Thanks in advance!

**Comment by apontzen** [comment]
Could you post a sample file (preferably small) that reproduces this problem?

**Comment by weiguangcui** [comment]
Please let me make a comment, the new AHF catalogue (with more columns) can be loaded without a problem, but it's always loaded as a dummy halo, even with the parameter 'dummy=False'. We think the problem is at where the halo links to the simulation. It is OK to upload the AHF catalogue, but the simulation is too large, several Gbs. Is it possible to share with you through Google drive or something?

**Comment by apontzen** [comment]
Ideally a small test simulation would be appreciated!

**Comment by apontzen** [comment]
(that way we can actually add a regression test for the new format)

**Comment by weiguangcui** [comment]
Hi Andrew, thank you.
I will ask @robmost to share the catalogue and simulation with you through email or private message.
Could you please contact with him (rob.mostoghiu@estudiante.uam.es)? or share your email address temporarily?
Thanks again for your kind helps

**Comment by robmost** [comment]
Hi,
I think we found the problem. Since I could load the '*AHF_halos' file but with no particle information we checked the `pynbody.halo._load_ahf_particles` class. 

Turns out the new AHF files I have were run with the AHF flag 'SUSSING2013' turned on. The '*AHF_particles' file had a different format, thus every halo ended up being a DummyHalo. Hopefully by re-running everything with this flag turned off everything will work as intended.
